docs/content/manual/manual.yml

```diff
@@ -2006,7 +2006,7 @@ sections:
 
           The `gmtime` builtin consumes a number of seconds since the
           Unix epoch and outputs a "broken down time" representation of
-          Greenwhich Meridian time as an array of numbers representing
+          Greenwich Mean Time as an array of numbers representing
```

Comment:
[Greenwich Mean Time](https://en.wikipedia.org/wiki/Greenwich_Mean_Time) is the time associated with https://en.wikipedia.org/wiki/Prime_meridian_(Greenwich)